esp-riscv-rt/src/lib.rs

```diff
@@ -672,7 +672,27 @@ _start_trap31:
 
 "#,
 r#"
-_start_trap: 
+_start_trap:
+"#,
+#[cfg(feature="fix-sp")]
+r#"
+    // move SP to some save place if it's pointing below the RAM
```

Comment:
Is this so we can actually run our hardfault handler and get some output?

Reply:
Yes that's the reason - do you think it needs more docs?

Reply:
I think its fine, was just double checking :)